engine/src/agents/agent.cpp

```diff
@@ -79,6 +79,10 @@ void Agent::perform_action()
     info_msg(*evalInfo);
     info_string(state->fen());
     info_bestmove(StateConstants::action_to_uci(evalInfo->bestMove, state->is_chess960()));
+    #ifdef MODE_STRATEGO
+    // Maybe use action_to_string because it is more general
```

Comment:
```suggestion
  
```
Sure it can be removed. It could be interesting to change the line above to be more general for games without a uci notation style. So that I do not need to specify Stratego as such a game

Reply:
Sure it can be removed. It could be interesting to change the line above to be more general for games without a uci notation style. So that I do not need to specify Stratego as such a game